text/_util.ts

```diff
@@ -0,0 +1,129 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+import { levenshteinDistance } from "./levenshtein_distance.ts";
+import { assert } from "../assert/mod.ts";
+
+// NOTE: this metric may change in future versions (e.g. better than levenshteinDistance)
+const getWordDistance = levenshteinDistance;
+
+/**
+ * get most-similar word
+ *
+ * @example
+ * ```ts
+ * import { closest } from "https://deno.land/std@$STD_VERSION/text/_util.ts";
+ *
+ * const possibleWords: string[] = ["length", "size", "blah", "help"];
+ *
+ * // case-insensitive by default
+ * const word = closest("hep", possibleWords);
+ * ```
+ *
+ * @param givenWord - The string to measure distance against
+ * @param possibleWords - The string-array that will be sorted
+ * @param options.caseSensitive - Flag indicating whether the distance should include case. Default is false.
+ * @returns A sorted copy of possibleWords
+ * @note
+ * the ordering of words may change with version-updates
+ * e.g. word-distance metric may change (improve)
+ * use a named-distance (e.g. levenshteinDistance) to
+ * guarantee a particular ordering
+ */
+export function closest(
```

Comment:
I feel like this is missing a word here. `closestWord`? Not sure.

Reply:
could be closestString since spaces are allowed

Reply:
`closestString` makes sense to me. I don't feel strongly either way, just brought this up for bikeshedding.

Reply:
done